### PR TITLE
Update affiliations for Ville and Paul

### PR DESCRIPTION
**Pull request by craigbox**
/kind documentation

(I guess)

### CODE REVIEW

**Review by evankanderson — COMMENTED** [review]
/lgtm
/approve

**Comment by vaikas** [comment]
/lgtm
/approve
/hold
Just a hold so @pmorie has a chance to look.

File: STEERING-COMMITTEE.md [hunk 1]
```diff
@@ -95,8 +95,8 @@ first name):
 | -------------------------------------------------------------- | ---------------- | ------------ | ---------------------------------------------------- | ---------- | --------
 | <img width="30px" src="https://github.com/thisisnotapril.png"> | April Kyle Nassi | Google       | [@thisisnotapril](https://github.com/thisisnotapril) | Bootstrap  | 2021     |
 | <img width="30px" src="https://github.com/mbehrendt.png">      | Michael Behrendt | IBM          | [@mbehrendt](https://github.com/mbehrendt)           | Bootstrap  | 2021     |
-| <img width="30px" src="https://github.com/pmorie.png">         | Paul Morie       | Independent  | [@pmorie](https://github.com/pmorie)                 | 2020-12-11 | 2022     |
-| <img width="30px" src="https://github.com/vaikas.png">         | Ville Aikas      | Independent  | [@vaikas](https://github.com/vaikas)                 | 2020-12-11 | 2022     |
+| <img width="30px" src="https://github.com/pmorie.png">         | Paul Morie       | Apple        | [@pmorie](https://github.com/pmorie)                 | 2020-12-11 | 2022     |
```
Comment (pmorie) [inline STEERING-COMMITTEE.md:98]: I'm not representing my employer, so I'd prefer to remain listed as independent. Thank you

**Pull request merged**